app/actors/hyrax/actors/collections_membership_actor.rb

```diff
@@ -81,7 +81,13 @@ def has_destroy_flag?(hash)
         # NOTE: Only called from create.  All collections are being added as parents.  None are being removed.
         def extract_collection_id(env, attributes_collection)
           return unless attributes_collection && attributes_collection.size == 1
-          env.attributes[:collection_id] = attributes_collection.first.second['id']
+          collection_id = attributes_collection.first.second['id']
```

Comment:
can you document what this is doing?  Maybe use yard_doc to describe `attributes_collection`, it's not clear what the shape of that data is.